### PR TITLE
Speed up CI runtime for R package

### LINKED ISSUE #55: Reduce CI runtime for R
At the moment most of the time spent in CI is due to R (building dependencies). @hongyuanjia do you know if we can use pre-compiled binaries? Or do you know of other ways to considerably reduce the build time for the R version?

### PR DESCRIPTION
**Pull request by hongyuanjia**
## Pull request overview

- Closes #55. Speed up CI runtime for R package via:
  * Using RStudio Package Manager to install packages via pre-compiled binaries
  * Do not install packages in `Suggests` field of devtools
  * Remove code coverage checking
- Related to #56 
  * ~Create `NAMESPACE` before calling `devtools::document()`~ Not necessary. `devtools::document()` will automatically create one. The root for the error lies in missing `RcppExports.R`
  * Manually call `Rcpp::compileAttributes()` to make sure `RcppExports.R` is created

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -29,14 +29,12 @@ matrix:
     - language: r
       r: release
       before_install:
-        - Rscript -e 'install.packages("devtools", dependencies = TRUE)'
+        - Rscript -e 'options(repos = c(CRAN = "https://demo.rstudiopm.com/all/__linux__/xenial/latest")); install.packages("devtools")'
```
Comment (dmey) [inline .travis.yml:32]: I am not sure this has improved the runtime -- the packages are still compiled from source.

**Comment by hongyuanjia** [comment]
> I would probably to keep this as is and eventually update this on CRAN at the new release.

I revise the Description because CRAN did not allow the Description to start with the package name `Psychrolib`. `R CMD check` will give a warning on CRAN machines.

File: .travis.yml (hunk 1 above)
Comment (hongyuanjia) [reply .travis.yml:32]: You are right. Not sure why it does not work on Travis. Will keep the original. But remove `dependencies = TRUE` did reduce the package sizes to install. So I will keep it.

**Comment by hongyuanjia** [comment]
The CI for R has been fixed. I will keep the current Description field as it is and update it on the next CRAN submission.

**Pull request merged**